docs/source/plugins/using_with_cli.md

```diff
@@ -0,0 +1,31 @@
+# Using Plugins with CLI
+
+:::{index} Using Plugins with CLI
+:::
+
+## Enabling Plugins for the CLI
+Plugin enablement does not persist between CLI runs, 
+so every CLI execution needs any used plugins to be specified each time.
+
+[//]: # (TODO: link below --plugins mentions to CLI documentation once it's implemented)
+
+___
+### By Name
+The plugin's name can be passed to the `--plugins` CLI option.
```

Comment:
Should also probably mention that multiple plugins can be specified by providing them as a `|` separated string. `python arelleCmdLine.py --plugins "myplugin1|myplugin2"  ...`